### PR TITLE
Swift 4 Upgrade

### PR DESCRIPTION
**Pull request by leesherwood**
This pull request gets SwiftIcons working on Xcode 9 & Swift 4. 

Majority of changes are swapping `NSXXXAttributeName` for the `NSAttributedStringKey` variants used in Swift 4.

_Note: I'm not 100% sure which (if any) of the project files should be included in pull requests, so i've taken a best guess based on other pull requests i've looked at on other repo's. If anything is missing, please let me know and i'll modify the commit & pull request._

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 os: osx
 language: swift
-osx_image: xcode8
+osx_image: xcode9
 
 branches:
     only:
```
Comment (ranesr) [inline .travis.yml]: Can you also please update the destination to 'platform=iOS Simulator,name=iPhone 8,OS=11.0' in line 10? Thanks.

Comment (leesherwood) [reply .travis.yml]: Done, and squashed into a single commit.

Comment (stockmind) [reply .travis.yml]: Would love to see it merged for easy cocoapods management!

**Comment by ranesr** [comment]
Thanks all!

**Pull request merged**